images/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("@rules_multirun//:defs.bzl", "command", "multirun")
```

Comment:
We have just a single command being run with multirun() at the moment. I assume the intention is that there will be more such commands in the future which would provide a benefit from using multirun?